clusterloader2/pkg/measurement/measurementmanager.go

```diff
@@ -0,0 +1,67 @@
+/*
```

Comment:
Since you're already inside measurement/ directory, I think you can simply name this file `manager.go` or sth.

Reply:
done